frontend_lib/README.md

```diff
@@ -55,7 +55,7 @@ Using the browser libary does not work: `tracim_frontend_lib` needs to be compat
 
 # Building a Tracim app using `tracim_fontend_lib`
 
-In the code of your app, use `require('tracim_fontend_lib')` or `import ... from tracim_fontend_lib` to import the library.
+In the code of your app, use `import ... from tracim_fontend_lib` to import the library (or `require('tracim_fontend_lib')`).
```

Comment:
does "requiring" tracim_frontend_lib works ?
It should, but since we don't use it that way in Tracim, I must admit I have never tested it

Reply:
I'll test